data/pages-content/jobs-board.md

```diff
@@ -392,3 +392,135 @@ Kiwi.com uncovers cheap travel options that others simply can't find.
+
+<details>
+  <summary><h4 id="energy-meteo1-full-stack-software-developer">Full Stack Software Developer</h4></summary>
+
+  We are looking for a full-stack software developer (f/m/d) for our headquarter in Oldenburg, Germany as soon as possible.
+
+  **Your task:**
+
+  We are technologically diverse. Python, Java and TypeScript are used as programming languages. You will develop software components for our SaaS products. Your possible areas of work could be:
+
+  - Infrastructure Tooling
+  - REST APIs (backend and frontend - API clients and web)
+  - GUI development for scientific applications and data visualization
+  - Database development and tuning
+  - Time series management
+  - Message queues / brokers (Kafka, RabbitMQ)
+  - Data pipelines
+  - Container Platform (Docker, Kubernetes)
+  - Microservices
+
+  **Your profile:**
+
+  You want to think and question, are willing to lend a hand and solve problems. These features make you stand out:
+
+  - You have completed a scientific degree in the field of computer science, software engineering, physics or mathematics or an apprenticeship as an IT specialist for application development (f/m/d) or have a comparable qualification - cross-entry is also possible.
+  - You are equally welcome as a career starter (f/m/d) or with professional experience.
+  -  You have experience in an object-oriented or multi-paradigm programming language such as Python or Java.
+  - You have the drive to build robust and secure software.
+  - With your convincing appearance and your open, friendly and constructive manner, you can score points with us in the team.
+
+  **What you can expect:**
+  
+  - You get the chance to work on meaningful projects in the field of renewable energies and to contribute your potential and ideas.
+  - Your working day is full of variety, your job is challenging and your team is motivated.
+  - Permanent employment contract - we are counting on you for the long term.
+  - Flexible working hours and the recording of every hour of overtime: You don't have to be on the spot at 7 o'clock in the morning. It is important to us that you show full commitment when you are there.
+  - Flat hierarchies and short decision-making processes - our leaders have open ears and doors.
+  - For the small break in between, there are drinks, organic fruit and snacks for free. For the big appetite, our in-house cafeteria awaits you with vegetarian, vegan and meat dishes made from regional products.
+  - The mood is also good outside our offices, e.g. at the annual company trip, team-building activities, Christmas parties, our monthly games night, and so on. Here we don't just talk about work.
+  - Other benefits: company pension scheme, bicycle leasing and bicycle vouchers or sports allowance.
+
+  You feel challenged and want to participate in shaping the energy transition with us?
+
+  Then apply by e-mail with your application documents: jobs@energymeteo.de
```

Comment:
duplicate to the next line?

Reply:
<img width="474" alt="Screenshot 2023-07-14 at 13 50 15" src="https://github.com/EuroPython/website/assets/44899477/b39cb323-c591-4e24-ae07-2f7ea48b8a0d">
I think, in shorter view "Apply via email" looks better than "Then apply by e-mail with your application documents", what do you think?

Reply:
absolutely!